src/content/en/tools/chrome-devtools/javascript/get-started.md

```diff
@@ -0,0 +1,274 @@
+probably location of the bug.
+
+[into]: /web/tools/chrome-devtools/images/step-into.png
+[over]: /web/tools/chrome-devtools/images/step-over.png
+
+## Step 4: Set another breakpoint
+
+Line-of-code breakpoints are the most common type of breakpoint. When
+you've got a specific line of code that you want to pause on, use a
+line-of-code breakpoint. Try it now:
+
+1. Look at the last line of code in `updateLabel()`, which looks like this:
+
+       `label.textContent = addend1 + ' + ' + addend2 + ' = ' + sum;`
+
+1. To the left of this code you can see the line number of this particular
+   line of code: **25**. Click on **25**. DevTools puts a blue icon on top
+   of **25**. This means that there is a line of code breakpoint on this line.
+   DevTools now always pauses before this line of code is executed.
+1. Click **Resume script execution** ![Resume script
+   execution][resume]{:.devtools-inline}. The script continues executing
+   until it reaches the line of code you placed the breakpoint on.
+1. Look at the lines of code in `updateLabel()` that have already executed.
+   DevTools prints out the values of `addend1`, `addend2`, and `sum`.
+
+The value of `sum` looks suspicious. It looks like it's being evaluated as
+a string, when it should be a number. This may be the cause of the bug.
+
+## Step 5: Check variable values
+
+Another common cause of bugs is when a variable or function produces
+a different value than expected. Many developers use `console.log()` to
+see how values change over time, but `console.log()` can be tedious and
+ineffective for two reasons. One, you may need to manually edit your code
+with lots of calls to `console.log()`. Two, you may not know exactly which
+variable is related to the bug, so you may need to log out many variables.
+
+One DevTools alternative to `console.log()` is Watch Expressions. Use
+Watch Expressions to monitor the value of variables over time.
+As the name implies, Watch Expressions aren't just limited to variables. You
+can store any valid JavaScript expression in a Watch Expression. Try it now:
+
+1. On the **Sources** panel of DevTools, click **Watch**. The section expands.
+1. Click **Add Expression** ![Add Expression][add]{:.devtools-inline}.
+1. Type `typeof sum`.
+1. Press <kbd>Enter</kbd>. DevTools shows `typeof sum: "string"`. The value
+   to the right of the colon is the result of your Watch Expression.
+
+As suspected, `sum` is being evaluated as a string, when it should be a
+number. This is the cause of the demo's bug.
+
+A second DevTools alternative to `console.log()` is the Console. Use the
+Console to evaluate arbitrary JavaScript statements.
+Developers commonly use the Console to overwrite the variable values
+when debugging. In your case, the Console can help you test out potential
+fixes for the bug you just discovered. Try it now:
+
+1. Press <kbd>Escape</kbd> to open the Console drawer. It opens at the bottom
```

Comment:
Some developers may already have the Console log open. So maybe reword this slightly to recognize this possibility.

Something like:  If the Console drawer isn't open, press Escape to open it at the bottom of your DevTools window.